internal/resolution/variablesources/installed_package.go

```diff
@@ -5,13 +5,15 @@ import (
 	"fmt"
 	"sort"
 
+	mmsemver "github.com/Masterminds/semver/v3"
```

Comment:
Yes, we have places like this. Example:

https://github.com/operator-framework/operator-controller/blob/205a48778e07f96819381269cb652536c6f895e6/internal/catalogmetadata/filter/bundle_predicates.go#L4-L5

`github.com/Masterminds/semver` was chosen for `Operator` API so users can specify version ranges for upgrades. `github.com/blang/semver` is used in catalog properties (in `olm.package.required`).

Reply:
Yikes.

Reply:
I added masterminds, and was hoping to drop blang, but there are ever-so-slight differences in acceptable syntax that we didn't get rid of blang because it processes and external, well-defined resource. We probably could get rid of blang if we convert the syntax.